etc/conky.profile

```diff
@@ -8,6 +8,8 @@ include globals.local
 
 noblacklist ${PICTURES}
 
+noblacklist /usr/lib/liblua*
```

Comment:
you could include allow-lua.inc.

Reply:
Yes, even if not all lines from `allow-lua.inc` are needed using includes is cleaner for maintenance than  custom `noblacklist` rules in profile.

Reply:
Well, I always try to keep the attack surface as small as possible. But if you prefer to add the inc file I'll change that.